### DISCUSSION TITLE
[Feedback] GitHub Mobile isn't support theme context and responsive in Profile Repo's README

### DISCUSSION THREAD

**Discussion by thaisonzdx** [community/community#34445] [Mobile]
1. Currently, GitHub Mobile app isn't displayed image by theme context correctly when I specify it by appending `#gh-dark-mode-only` or `#gh-light-mode-only` to the end of image url.
2. I also want to make a readme can responsive on mobile device, how to do this. This is my profile [https://github.com/sownt](https://github.com/sownt)

**Comment by Iron-Ham** [comment]
Hi @sownt

Thank you for reporting this! A few questions for you: 

- Are you running into this on iOS or Android?
- What version of the app are you currently using?

Thanks!

**Reply by thaisonzdx** [reply]
I am using GitHub Mobile 1.82.0 on Android 13 (Pixel 5)

**Reply by Iron-Ham** [reply]
Thanks for that @sownt! I've opened up an internal tracking issue for us to investigate further.

**Reply by ajmeese7** [reply]
I'm experiencing the same issue on a GitHub Mobile 1.84.0, Android 13, Pixel 6 Pro. Any fallbacks in the intermediary until this is resolved would be appreciated!

You can see this in action for the [meeseOS](https://github.com/meeseOS/meeseOS/blob/master/README.md) repository, the images display perfectly on desktop but in the mobile app all I see is the alt text.

**Comment by Kocins** [comment]
Ok
